lib/web_ui/lib/src/engine/text/line_breaker.dart

```diff
@@ -376,31 +374,25 @@ List<LineBreakFragment> _computeLineBreakFragments(String text) {
     // Do not break between closing punctuation and a nonstarter, even with
     // intervening spaces.
     // LB16: (CL | CP) SP* × NS
-    if ((prev1 == LineCharProperty.CL ||
-            baseOfSpaceSequence == LineCharProperty.CL ||
-            prev1 == LineCharProperty.CP ||
-            baseOfSpaceSequence == LineCharProperty.CP) &&
+    //
+    // The above is a quote from unicode.org. In our implementation, we did the
+    // following modification: Allow breaks when there are spaces.
```

Comment:
Do we have details about why we modified it documented anywhere? Maybe expand on it here a bit?

Reply:
Good point. We modified it to match Chrome's behavior (which makes more sense to me than the unicode spec in these cases). I'll explain it in the comment.